tutorials/telegraf-output-plugin.md

```diff
@@ -0,0 +1,346 @@
+# Collecting metrics with the PostgreSQL and TimescaleDB output plugin for Telegraf
+
+Telegraf can collect metrics from a wide array of inputs and write them to a wide array of outputs. 
+It is plugin-driven for both collection and output of data so it is easily extendable. 
+It is written in Go, which means that it is compiled and standalone binary that can be executed on any system with no need for external dependencies, or package management tools required. 
+
+Telegraf is an open-source tool. It contains over 200 plugins for gathering and writing different types of data written by people who work with that data. 
+
+We wrote the PostgreSQL output plugin which also has the ability to send data to an TimescaleDB hypertable. The [pull request][pull-request] is open and currently under review by the Telegraf developers, waiting to be merged. To give developers the opportunity to try this functionality, we built downloadable binaries of Telegraf with our plugin already included.
+
+This tutorial will run through a couple of examples on how to use the PostgreSQL/TimescaleDB output plugin for Telegraf.
+
+## Installation [](telegraf-installation)
+
+### Before we start
+
+Before we start, you will need [TimescaleDB installed][getting-started] and a means to connect to it.
+
+### Setting up Telegraf
+
+Telegraf is written in Go, and the current build process of the tool is configured to produce one standalone binary. Because of this all the code for the different plugins must be part of that binary. We have an unofficial build of Telegraf version 1.10.4 with our plugin added. You can download from the following links: 
+
+* Linux amd64: [:DOWNLOAD_LINK: `deb`][deb-build] [:DOWNLOAD_LINK: `rpm`][rpm-build] [:DOWNLOAD_LINK: `binary`][linux-bin-build]
+* Windows amd64 [:DOWNLOAD_LINK: `binary/exe`][windows-build]
+* MacOS amd64 [:DOWNLOAD_LINK: `binary`][mac-build]
+
+We can also provide you with builds for:
+
+* Windows i386
+* Linux (i386, armhf, armel, arm64, static_amd64, s390x, mipsel)
+* FreeBSD (amd64, i386)
+
+if you contact us in our [community slack][public-slack]
+
+Once you download the binary and extract it to a suitable location (or install the packages) we can test out the build. 
+You may have to make the file executable by running `chmod +x telegraf`.
+We can check the version of the installed Telegraf with
+
+```
+$ telegraf --version
+```
+
+If the installation was good it should print out `Telegraf 1.10.4-with-pg`.
```

Comment:
is good the right adjective here? i think successful seems more appropriate @solugebefola

Reply:
I like successful 👍